### PR TITLE
add nextjs example

### PR DESCRIPTION
**Pull request by sw-yx**
## What was changed
add nextjs example

## Why?
pple want the nextjs

image

### CODE REVIEW

**Comment by sw-yx** [comment]
testing out of scope for this example - Val will be working on our docs for that one

File: nextjs-oneclick/pages/api/workflow.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+  // pass options to the Connection constructor to configure TLS and other settings.
+  const connection = new Connection();
+  // Workflows will be started in the "default" namespace unless specified otherwise
+  // via options passed the WorkflowClient constructor.
+  const client = new WorkflowClient(connection.service);
+  // Create a typed handle for the OneClickBuy Workflow.
+  const workflow = client.createWorkflowHandle(OneClickBuy, { taskQueue: 'tutorial' });
+  const runId = await workflow.start('Temporal');
+  console.log('0', runId);
+  console.log('1', await workflow.query.purchaseState())
```
Comment (vkarpov15) [inline nextjs-oneclick/pages/api/workflow.js:15]: Missing semi-colons?

File: nextjs-oneclick/temporal/lib/activities.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+"use strict";
+Object.defineProperty(exports, "__esModule", { value: true });
+exports.canceledPurchase = exports.checkoutItem = void 0;
+// @@@SNIPSTART nodejs-hello-activity
```
Comment (vkarpov15) [inline nextjs-oneclick/temporal/lib/activities.js:4]: Get rid of this snip before merging please. Also, let's not commit compiled TS code.

File: nextjs-oneclick/temporal/src/activities.ts [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// @@@SNIPSTART nodejs-hello-activity
```
Comment (vkarpov15) [inline nextjs-oneclick/temporal/src/activities.ts:1]: Get rid of this snip please, this isn't a hello world example :)

File: nextjs-oneclick/temporal/src/worker.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
+// @@@SNIPSTART nodejs-hello-worker
```
Comment (vkarpov15) [inline nextjs-oneclick/temporal/src/worker.ts:1]: Same, get rid of this snip please.

**Pull request merged**